### DISCUSSION TITLE
All files deleted in share directory automatically in windows servers

### DISCUSSION THREAD

**Discussion by TestCoper** [gravitational/teleport#46949] [Q&A]
Dear GoTeleport Support Team,

I hope this message finds you well.

We are experiencing a critical issue with our Windows Server shared folders. Specifically, when we open an Excel file directly from a shared folder, all the files within that folder are automatically deleted immediately after the Excel file is opened. This issue occurs exclusively with Excel files; other types of data and files do not trigger this problem.

This unexpected deletion of files is causing significant disruption to our operations. We urgently need assistance to identify the cause and implement a solution to prevent further data loss.

Could you please help us investigate this issue as soon as possible?

Thank you for your prompt attention to this matter.

Best regards

**Comment by webvictim** [comment]
Hi - if you're a Teleport Enterprise customer, please open a Zendesk ticket for assistance. Support here on Github is best effort only.

**Reply by TestCoper** [reply]
hi, im an open source user and your team didnt help me in zendesk :)

**Comment by webvictim** [comment]
What size are the Excel files in question here?

Do you see any errors in the top right of the browser tab?

![image](https://github.com/user-attachments/assets/7c94127a-34df-418a-a37e-d9058c062a4f)

If so, what do they say?

**Reply by zmb3** [reply]
I'd check the audit log too just to see if there are any interesting audit events.
